localization/english/wc_traits_l_english.yml

```diff
@@ -536,3 +536,10 @@
  TRAIT_FLAG_DESC_frostmourne_interactions:0 "Owner of [frostmourne|E] is less likely to refuse [interactions|E]#!"
 
  TRAIT_FLAG_DESC_no_aging_penalties:0 "@warning_icon!#X Is not aging#!"
+
+ trait_aspect_of_infinite:0 "Aspect of Infinite"
+ trait_aspect_of_infinite_desc:0 "The Aspect of Infinite was once the powerful Aspect of Time, chosen by the titan Aman'thul to govern the flow of time and the threads of destiny. However, after seeing a vision of his own death, his fear consumed his mind and soul, leading him to turn against his own kind and seek control over time itself. Corruption overtook him, and now he commands powerful servants, seeking to rewrite history to prevent his own death and posing a grave threat to the timeline and the world itself."
```

Comment:
```suggestion
 trait_aspect_of_infinite_desc:0 "The Aspect of Infinity was once the powerful Aspect of Time, chosen by the titan Aman'thul to govern the flow of time and the threads of destiny. However, after seeing a vision of his own death, fear consumed his mind and soul, leading him to turn against his own kind and seek control over time itself. Corruption overtook him, and now he commands powerful servants, seeking to rewrite history to prevent his own death and posing a grave threat to the timeline and the world itself."
```